app/screens/edit-competition/page.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+type Props = {}
+
+const EditCompetition = (props: Props) => {
```

Comment:
documentation... pretty much every screen or comp needs it unless we are going to delete it. Even then documentation explaining what it is and why we might delete it later is great